doc/nrf/ug_bt_mesh_discoverable_dm_app.rst

```diff
@@ -0,0 +1,44 @@
+.. _ug_bt_mesh_discoverable_dm_app:
+
+Discovering Bluetooth mesh device in nRF Connect Device Manager
+###############################################################
+
+If your Bluetooth mesh device is not visible in the list of available devices when using the `nRF Connect Device Manager`_ mobile app (for example when performing FOTA updates), it is because the device is not advertising the SMP service UUID and the filter that only shows devices advertising this service is enabled.
+The device can still be discovered through a service discovery, for example using the `nRF Connect for Mobile`_ app.
+
+To make sure your device is visible in the `nRF Connect Device Manager`_ mobile app, do one of the following:
+
+* Disable the filter that only shows devices advertising the SMP service UUID in the `nRF Connect Device Manager`_ mobile app.
+* Make the device advertise the SMP service UUID, and thus be discoverable by the `nRF Connect Device Manager`_ mobile app with the filter enabled.
+
+Disabling the filter
+********************
+
+To disable the filter in the `nRF Connect Device Manager`_ mobile app, do the following steps:
+
+1. Tap the :guilabel:`Filter` button at the right top corner of your screen.
+#. Deselect :guilabel:`Only devices advertising SMP UUID`.
+
+You should see the device node appear in the list of devices.
+
+Advertising SMP UUID
+********************
+
+To make sure that your Bluetooth mesh device advertises the SMP service UUID, you can use the :ref:`bluetooth_mesh_light` sample where this is enabled through configuration overlay file :file:`overlay-dfu.conf`.
+Alternatively, do the following:
+
+1. Enable and configure the extended advertising API for your Bluetooth mesh sample.
+   To do so, add the following to the :file:`prj.conf` file of your sample::
+
+    CONFIG_BT_EXT_ADV=y
+    CONFIG_BT_EXT_ADV_MAX_ADV_SET=2 # 1 set for mesh, 1 set for advertising SMP server
+    CONFIG_BT_MAX_CONN=2 # 1 conn for GATT Proxy/PB-GATT, 1 conn for SMP
+
+	.. note::
+          As of |NCS| v2.1.2, the extended advertising API is enabled (CONFIG_BT_EXT_ADV=y) by default.
+          You would still need to configure CONFIG_BT_EXT_ADV_MAX_ADV_SET and CONFIG_BT_MAX_CONN as defined above.
+
+#. Use the code from the :ref:`zephyr:smp_svr_sample` sample to advertise SMP UUID.
+
+This will make the device node discoverable by the `nRF Connect Device Manager`_ mobile app with the :guilabel:`Only devices advertising SMP UUID` filter enabled.
+Observe that the device node appears in the list of devices in the mobile app.
```

Comment:
```suggestion
This will make the device discoverable by the `nRF Connect Device Manager`_ mobile app with the :guilabel:`Only devices advertising SMP UUID` filter enabled.
Observe that the device appears in the list of devices in the mobile app.
```